### ISSUE TITLE
"Running test"... takes up a lot of space

### ISSUE THREAD

**Issue by mitchellwrosen** [ndmitchell/ghcid#167]
I'm running ghcid with a `-T thing -W`, where `thing` is a function that prints no output:

    > thing
    >

However, ghcid gives a ton of screen real estate for this test for some reason. My terminal currently looks like:

     18 | import Data.Text
        | ^^^^^^^^^^^^^^^^
    Running test...

    ...done

The warning(s) here were mostly pushed off of the screen by `thing` running! If I don't pass `-W`, I see them all, of course.

Any idea what might be going on? Thanks!

**Comment by ndmitchell** [comment]
It's deliberate. I try and clear the screen before running a test, but I can't remember exactly why. I'll have to take a look at the code for any hints...

**Comment by ndmitchell** [comment]
I remember - you want the warning message (or All good) message to appear at the top of the screen, and you want to clear any previous stale messages that might be around still. The only way that ghcid has to do that is emit a page full of blank lines and leave the cursor at the bottom.

I'll try using cursorUp from the ANSI terminal and see if that helps.

**Comment by codygman** [comment]
This space messes up following the newest output in emacs `compile` mode because it works by always going to the end of the file. Is your use case something like that @mitchellwrosen ?

It would be nice to have a flag to disable emitting those spaces... or maybe I just need some more custom elisp to find the last non-space character :)

**Comment by mitchellwrosen** [comment]
@codygman No, my use case was just the normal ghcid rapid feedback loop, plus re-running the application (via the repl) on every save
